packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -723,6 +723,44 @@ export class DataStores implements IDisposable {
 		}
 	}
 
+	/**
+	 * This is called to delete unused nodes.
```

Comment:
Some comments on the return type and what it should contain would be useful. It's not very apparent why return the list at all.